### ISSUE TITLE
webview.get_current_url() not working properly with Qt on GNU/Linux

### ISSUE THREAD

**Issue by adbenitez** [r0x0r/pywebview#198]
`webview.get_current_url()` is returning None even when a page have been loaded.
I am using GNU/Linux and PyQt5(PyQt5-5.10-5.10.0-cp35.cp36.cp37-abi3-manylinux1_x86_64.whl)
I tried `get_current_url` with PyQt5(PyQt5-5.10-5.10.0-cp35.cp36.cp37-none-win_amd64.whl) on Windows and it worked.

**Comment by shivaprsdv** [comment]
Can't reproduce in PyQt 5.9.1. Can you check? It could help pin-point if it is a
5.10 only issue.

**Comment by adbenitez** [comment]
mmm... bad news `get_current_url.py` returns `None` webview.gtk too :(

**Comment by adbenitez** [comment]
correction:  sorry, `get_current_url.py` works fine using gtk (it was returning `None` due to low Internet connection), but returns fine when the page has loaded. In the case of Qt I found that the first time I call `get_current_url` return `None` but if I call the function again in the same program it return the current url, this happens even if I use `time.sleep(10)` and load `file:///` so  the page load faster.
I had installed `PyQt5-5.9.1-5.9.2-cp35.cp36.cp37-abi3-manylinux1_x86_64.whl`
and had the same problem the fist time I call `get_current_url`  returns `None`.
